components/bio-formats-plugins/src/loci/plugins/in/importer-options.txt

```diff
@@ -189,9 +189,19 @@ default = false
 type = boolean
 label = Display_ROIs
 info = <b>Display ROIs</b> - \
-  Adds any ROIs in the file to ImageJ's ROI manager.
+  Adds any ROIs in the file to ImageJ.
 default = false
 
+[roiMode]
+label = ROIs_Import Mode:
+info = <b>ROI Import Mode</b> - \
+  ROIs can be imported to the ROI manager or an overlay.
+type = string
+default = ROI Manager
```

Comment:
@stelfrich: thinking mostly in terms of maintenance, imposing a consistent naming scheme might minimize similar issues if the new `roiMode` gets reused anywhere else so I would vote for that.